.gitignore

```diff
@@ -1,3 +1,5 @@
 /obj
 walld
-.TODO
\ No newline at end of file
+.TODO
+/builddir
```

Comment:
For the record, since Meson 0.57.0 you will automatically have a file `builddir/.gitignore` created that ignores `*` for you; this is done because:
- the configured build directory can be named anything, and it's not something you can predict in advance to check into your gitignore
- on the other hand, Meson always knows what it's called, because it must create it, and Meson also knows that generated files should never be checked into git -- so there's no reason not to do this automatically for you, so that you don't have to think about it yourself. :)